File: exercicios em Python/q70.py
import os

soma = 0
result = 0
print("\nEste programa determina o faturamento mensal de um conta de energia.")
n = int(input("\nQuantas aparelhos com gasto de energia voce possui em sua casa: "))
i = 1
while i <= n:
        print("\n\t\t",i," Aparelho!")
        el = float(input("\nQuantas unidade do eletronico voce possui: "))
        w = float(input("\nDigite a quantidade de W o aparelho gasta: "))
        hs = float(input("\nDigite a quantidade de horas voce gasta com o aparelho ligado: "))
        result = int((el * w * hs * 30) / 1000)
        soma = int(result + soma)
        i = i + 1

print("\nA fatura do mes de energia equivale a: ",soma)
print("\nObrigado por utilizar nosso sistema!")
os.system("pause")

File: exercicios em C/q70.cpp
#include <stdio.h>

int main(void){
	int i, n, result, kwh, el, w, hs, soma;
	soma = 0;
	result = 0;
	printf("\n\nEste programa determina o faturamento mensal de um conta de energia.");
	printf("\n\nquantos eltronicos possui em sua casa: ");
	scanf("%d", &n);
	for (i = 1; i <= n; i++){
		printf("\n\nQuantas unidade do %d eletronico voce possui: ", i);
		scanf("%d", &el);
		printf("\nDigite a quantidade de W o %d aparelho gasta: ", i);
		scanf("%d", &w);
		printf("\nDigite a quantidade de horas voce gasta com %d aparelho ligado: ", i);
		scanf("%d", &hs);
		result = (el * w * hs * 30) / 1000;
		soma = result + soma;
	}
	printf("\n\nA fatura do mes de energia equivale a: %d", soma);
	printf("\n\nObrigado por utilizar nosso sistema!");
	return 0;
}
